package.json

```diff
@@ -26,7 +26,11 @@
   "devDependencies": {
     "all-contributors-cli": "^6.26.1",
     "concurrently": "^8.2.1",
+    "husky": "^8.0.0",
     "prettier": "3.1.0",
-    "husky": "^8.0.0"
+    "tailwindcss": "^3.4.1"
```

Comment:
tailwind css is already installed in the client directory, no need to install it globally